.changes/mac-fullscreen-resize.md

```diff
@@ -0,0 +1,6 @@
+---
+"tao": patch
+---
+
+Fix resize event emits before fullscreen actually exit.
```

Comment:
doesn't this seem counter-intuitive? I'd expect the resize event to happen after fullscreen exists.

Reply:
Winit has same issue, `window_did_resize` is called after `window_will_exit_fullscreen` but before `window_did_exit_fullscreen`.

Reply:
alright, lets merge this and see later how we can port to winit